docs/utilities/parameters.md

```diff
@@ -161,20 +161,19 @@ The AWS Systems Manager Parameter Store provider supports two additional argumen
 | **decrypt**   | `False` | Will automatically decrypt the parameter.
 | **recursive** | `True`  | For `get_multiple()` only, will fetch all parameter values recursively based on a path prefix.
 
-> **Example**
+???+ example
```

Comment:
@heitorlessa  i'm on the fence with using this admonition because on other places like tracer.md we use quote. However in parameters.md  we have few example blocks ... but thinking loud ... every snippet is an example ...

Reply:
Every snippet would be an example ;) Also, notice how admonition decreases the quality of the text (low eyesight accessibility issue) when using this approach -- I've reverted in places I noticed this

![image](https://user-images.githubusercontent.com/3340292/147748452-ae3a1252-7e06-4190-b5f7-ee6f85bba1d3.png)

Reply:
We can be more explicit using the new `title=` option -- I'll change all those single block snippets to use `title` instead of the tabbed content.

![image](https://user-images.githubusercontent.com/3340292/147748628-aed689f4-6dfd-472b-88fc-c8ad7a413881.png)